AirCasting/RootAppView.swift

```diff
@@ -41,21 +43,24 @@ struct RootAppView: View {
     }
 
     var mainAppView: some View {
-            MainTabBarView(measurementUpdatingService: DownloadMeasurementsService(
-                            authorisationService: userAuthenticationSession,
-                            persistenceController: persistenceController,
-                            baseUrl: dependancies.urlProvider), urlProvider: dependancies.urlProvider, measurementStreamStorage: measurementStreamStorage, sessionSynchronizer: sessionSynchronizer, sessionContext: CreateSessionContext())
-                .environmentObject(dependancies.bluetoothManager)
-                .environmentObject(userAuthenticationSession)
-                .environmentObject(persistenceController)
-                .environmentObject(dependancies.networkChecker)
-                .environmentObject(lifeTimeEventsProvider)
-                .environmentObject(userSettings)
-                .environmentObject(locationTracker)
-                .environmentObject(userRedirectionSettings)
-                .environmentObject(dependancies.airBeamConnectionController)
-                .environment(\.managedObjectContext, persistenceController.viewContext)
-        }
+        let sessionStoppableFactory = SessionStoppableFactoryDefault(microphoneManager: microphoneManager,
+                                                                     measurementStreamStorage: measurementStreamStorage,
+                                                                     synchronizer: sessionSynchronizer)
+        return MainTabBarView(measurementUpdatingService: DownloadMeasurementsService(
+                                authorisationService: userAuthenticationSession,
+                                persistenceController: persistenceController,
+                                baseUrl: dependancies.urlProvider), urlProvider: dependancies.urlProvider, measurementStreamStorage: measurementStreamStorage, sessionStoppableFactory: sessionStoppableFactory, sessionSynchronizer: sessionSynchronizer, sessionContext: CreateSessionContext())
```

Comment:
Maybe we can stuck this vertical to be more readable - what is your opinion about that?